compiler/luci/pass/src/FuseActivationFunctionPass.cpp

```diff
@@ -42,6 +42,11 @@ bool fuse_activation_function(luci::CircleNode *node)
   // This will skip fuse for concat as luci-interpreter doesn't support this yet
   if (dynamic_cast<luci::CircleConcatenation *>(pred_node) != nullptr)
     return false;
+  // TODO remove this work-around
+  // This will skip fuse for TransposeConv as backends does not support this yet
+  // NOTE remove this when XpSepActFromTransposeConvOpPass is removed
```

Comment:
May I ask what is `XpSepActFromTransposeConvOpPass`?

Reply:
Look inside ; #11387 .
`XpSepActFromTransposeConvOpPass ` will run for default to separate any fused activation from TransposeConv for the mean time.